src/Stripe.net/Services/Subscriptions/SubscriptionCreateOptions.cs

```diff
@@ -5,8 +5,14 @@ namespace Stripe
     using Newtonsoft.Json;
     using Stripe.Infrastructure;
 
-    public class SubscriptionCreateOptions : SubscriptionSharedOptions
+    public class SubscriptionCreateOptions : BaseOptions, IHasMetadata
     {
+        /// <summary>
+        /// A non-negative decimal between 0 and 100, with at most two decimal places. This represents the percentage of the subscription invoice subtotal that will be transferred to the application owner’s Stripe account. The request must be made with an OAuth key in order to set an application fee percentage. For more information, see the application fees <see href="https://stripe.com/docs/connect/subscriptions#collecting-fees-on-subscriptions">documentation</see>.
```

Comment:
Flagging it'd be good to "reformat" the comments to be with the max line size. But not a blocker